Hackathons.md

```diff
@@ -4,6 +4,9 @@
 * [Major Leage Hacking]
 * [Devpost]
 
+#### Boilerplate
+# [Node.js Hackathon Starter Kit]
```

Comment:
This appears as a headline (check View to confirm). '#' in Markdown is a heading. You meant to put an asterisk ('*').